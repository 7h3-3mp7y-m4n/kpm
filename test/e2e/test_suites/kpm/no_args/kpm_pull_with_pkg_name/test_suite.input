kpm pull k8s