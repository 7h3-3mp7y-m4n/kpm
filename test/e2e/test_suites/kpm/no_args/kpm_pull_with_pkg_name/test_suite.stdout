kpm: start to pull 'k8s'.
kpm: the lastest version '1.27.2' will be pulled.
kpm: pulling 'kusionstack/k8s:1.27.2' from 'ghcr.io/kusionstack/k8s'.
kpm: pulled 'k8s' in '<workspace>/ghcr.io/kusionstack/k8s' successfully.
